extra/1-currency-conversion.js

```diff
@@ -15,7 +18,10 @@ function convertToUSD() {}
   They have also decided that they should add a 1% fee to all foreign transactions, which means you only convert 99% of the £ to BRL.
 */
 
-function convertToBRL() {}
+function convertToBRL(Pound) {
+  const BRL = 0.99 * Pound * 5.7;
```

Comment:
Nice, but you could think about breaking this out to improve readability. Your code should read like a story, so when you are breaking out and finding the 0.99 - how can you inform collaborators that's what you're doing? e.g. add a line with an additional variable such as priceAfterFee

Reply:
> Nice, but you could think about breaking this out to improve readability. Your code should read like a story, so when you are breaking out and finding the 0.99 - how can you inform collaborators that's what you're doing? e.g. add a line with an additional variable such as priceAfterFee

Done!t Thank you